rumqttc/examples/websocket.rs

```diff
@@ -0,0 +1,47 @@
+use tokio::{task, time};
+
+use rumqttc::{self, AsyncClient, MqttOptions, QoS, Transport};
+use std::error::Error;
+use std::time::Duration;
+
+#[tokio::main(worker_threads = 1)]
+async fn main() -> Result<(), Box<dyn Error>> {
+    pretty_env_logger::init();
+
+    // port parameter is ignored when scheme is ws  
+    let mut mqttoptions = MqttOptions::new("clientId-aSziq39Bp3", "ws://broker.mqttdashboard.com:8000/mqtt", 8000);
+    mqttoptions.set_transport(Transport::Ws);
```

Comment:
How about adding a feature gated panic here to inform the user of the need to use `--features=websocket`?

Reply:
@IniterWorker?